templates/master.yaml.tmpl

```diff
@@ -2370,8 +2370,10 @@ storage:
                 - /hyperkube
                 - apiserver
                 - --allow-privileged=true
+                - --anonymous-auth=false
                 - --kubelet-https=true
                 - --secure-port=443
+                - --insecure-port=0
```

Comment:
Usually 0 is a randomly used port. Isn't that the case?

Reply:
No, in this case 0 is disabled